ton/src/tep/tvm_results/tvm_result.rs

```diff
@@ -39,4 +41,47 @@ mod trait_impl {
     impl TVMResult for TonAddress {
         fn from_stack(stack: &mut TVMStack) -> TonResult<Self> { Ok(TonAddress::from_cell(&stack.pop_cell()?)?) }
     }
+
+    impl TVMResult for MetadataContent {
+        fn from_stack(stack: &mut TVMStack) -> TonResult<Self> { Ok(MetadataContent::from_cell(&stack.pop_cell()?)?) }
+    }
+
+    impl TVMResult for TLBCoins {
```

Comment:
I have some concerns about that, need to discuss

Likely we should use `Coins` type instead of `TLBCoins` for TVMResult entities